docs/BulkExport.md

```diff
@@ -42,16 +42,17 @@ For more details on Bulk Export, see the [Azure API for FHIR Export Data page](h
 
 Below are set of additional query parameters users can specify in addition to ones defined in the Azure API For FHIR documentation
 1. \_format in FHIR Server:  \_format allows a user to select a format for the file structure that the export job creates. Different formats can be defined in the appSettings by combining constants, folder level breaks ('/'), and known tags. The tags will be replaced with data when the job is run. The three supported tags are: 
-* **resourcename**: replaces with the resource type being exported
-* **timestamp**: replaces with a timestamp of the job's queried time
-* **id**: replaces with the GUID of the export job
+* **resourcename**: Replaced with the resource type being exported.
+* **timestamp**: Replaced with a timestamp of the job's queried time.
+* **id**: eplaced with the GUID of the export job.
+  
 1. \_max_count:  \_max_count allows to reduce the number of resources exported by a single job. Users can use the _maxCount=xxxx query parameter or set MaximumNumberOfResourcesPerQuery in the export configuration section. The default is 10,000. Export operation needs memory to serialize the data when it is writing to the lake. To reduce out of memory exceptions due to additional memory, user can choose to reduce the _max_count value by decrements of 1000. It would be beneficial for user to increase the compute memory on FHIR server as well.
 
 To use the format, you will need to set the following settings in the appSettings:
 
 | appSetting | Description | Example Value|
 |------------|-------------|--------------|
-| FhirServer:Operations:Export:Formats:#:Name | Name of the format you plan to call. The # should be replaced as you can specify multiple formats. We provide default values for 0 and 1 (for when a container is specified and when a container is not specified) so recommend starting with 2 | TestFormat |
+| FhirServer:Operations:Export:Formats:#:Name | Name the format you plan to call. The # should be replaced as you can specify multiple formats. We provide default values for 0 and 1 (for when a container is specified and when a container is not specified), so we recommend starting with 2 | TestFormat |
```

Comment:
`of` was removed here and I don't think it should have been. "Name of the format you plan to call" sounds better to me. :)